project_extra/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Project Extra
```

Comment:
@seb-elico , this PR is just for fix travis of all the modules in this repo , and this rst file is added for that reason , I didn't add any new module in this PR.